src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -3402,7 +3403,16 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
             notations.tag(m_xml, a);
             ornaments.tag(m_xml);
             m_xml.tagRaw(mxmlOrnam);
-            // accidental-mark is missing
+            if (a->isOrnament()) {
+                const Ornament* ornam = toOrnament(a);
+                for (size_t i = 0; i < ornam->accidentalsAboveAndBelow().size(); ++i) {
+                    Accidental* accidental = ornam->accidentalsAboveAndBelow()[i];
```

Comment:
Wouldn't it be possible to do the following?
```suggestion
                for (const Accidental* accidental : ornam->accidentalsAboveAndBelow()) {
```

Reply:
Way better and cleaner approach. Thanks for the hint!